tests/test_other.py

```diff
@@ -199,6 +199,15 @@ def test_emcc_output_worker_mjs(self):
     with open('hello_world.worker.js') as f:
       self.assertContained('import(', f.read())
 
+  def test_export_es6_implies_modularize(self):
+    self.run_process([EMCC, path_from_root('tests', 'hello_world.c'), '-s', 'EXPORT_ES6=1'])
+    src = open('a.out.js').read()
```

Comment:
I think you should be using `with open(...)` pattern like in other tests (see the one right above) to ensure that file is properly closed.

Reply:
Thanks, that's a better way. The PR has been updated :thumbsup: